package.json

```diff
@@ -23,7 +23,10 @@
   },
   "homepage": "https://github.com/tarunsinghdev/bird-app#readme",
   "dependencies": {
+    "cookie-parser": "^1.4.4",
+    "cors": "^2.8.5",
     "express": "^4.17.1",
+    "express-validator": "^6.2.0",
```

Comment:
mongoose has inbuilt validation, in the schema itself, we could add properties like `minLength: 4` and supports custom validation as well https://mongoosejs.com/docs/validation.html.
But custom validation for checking the correctness of email and password or credit card details would be tricky when done with regex therefore we'll use the https://www.npmjs.com/package/validator.
Therefore, though express-validator is a wrapper around validator package and provides some more features like sanitization and validation but that all can be done with validator package and mongoose inbuilt validation.
So that we would not bloat our application if we install express-validator whose features we don't actually use.

So better to uninstall express-validator and install validator only. Rest you can read more in the link provided.
Please make sure to correct these, I'm merging your request now.

Reply:
Will change express-validator with validator in future PR.